numba/config.py

```diff
@@ -182,7 +180,7 @@ def avx_default():
                 # XXX: need a dedicated PerformanceWarning class?
```

Comment:
Now that you have added a PerformanceWarning class, you can remove this comment.